src/index.html

```diff
@@ -5,10 +5,56 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo__link" href="index.html">
+        <img class="logo-img"
+        src="./images/logo.png"
+        alt="logo_image"
+        >
```

Comment:
You need to check and fix it everywhere
![image](https://github.com/mate-academy/layout_moyo-header/assets/64750958/39634030-52a1-481e-a00e-69e7973a672f)